app/src/main/java/org/simple/clinic/editpatient/EditPatientUpdate.kt

```diff
@@ -56,7 +56,7 @@ class EditPatientUpdate(
       is AlternativeIdChanged -> next(model.updateAlternativeId(event.alternativeId))
       is BpPassportsFetched -> dispatch(DisplayBpPassportsEffect(event.bpPasssports))
       is InputFieldsLoaded -> dispatch(SetupUi(event.inputFields) as EditPatientEffect)
-      is ColonyOrVillagesFetched -> next(model.updateColonyOrVillagesList(event.colonyOrVillages))
+      is ColonyOrVillagesFetched -> next(model.updateColonyOrVillagesList(event.colonyOrVillages), event.colonyOrVillages?.let { ShowColonyOrVillagesListEfect(it) } as EditPatientEffect)
```

Comment:
If we are saving it to the model already, why not use `ViewRenderer`? The effect way makes sense if we are not saving it to the model. Since you're saving it to the model, let's use the `ViewRenderer` for updating the autocomplete adapter, that way it will also be consistent across config changes and we don't have to make the DB call when the model already contains the names.